.github/workflows/pull-request.yaml

```diff
@@ -2,14 +2,19 @@ name: Lint, Test, and Danger
 
 on: pull_request
 
+# https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+permissions:
+  checks: write
+  pull-requests: write
+
 jobs:
   lint-test-danger:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repo
         uses: actions/checkout@v3
         with:
-          token: ${{ secrets.PAT_DECRAPIFIER }}
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is going to prevent duti from pushing up commits when finds automatic formatting or lint fixes but doesn't look like we're running those in CI right now anyway.